web/modules/custom/portland/modules/portland_address_verifier/js/addressVerifierView.js

```diff
@@ -66,13 +60,6 @@ AddressVerifierView.prototype.renderAddressVerifier = function () {
     // this._handlePostback();
 };
 
-AddressVerifierView.prototype._checkIfVerificationRequired = function () {
```

Comment:
All the client-side code related to checking whether the `location_verification_status` field is required has been removed. This is because a hidden input field doesn't have the required attribute set on it by the Drupal form API, so the client has no way to know. 

Instead, we detect this server-side and set a boolean in `drupalSettings` same as how other properties are passed.